test/test_utils.rb

```diff
@@ -1,6 +1,21 @@
 require 'helper'
 
 class TestUtils < JekyllUnitTest
+  context "The \`Utils.deep_merge_hashes\` method" do
```

Comment:
It looks like it does: https://github.com/seattlerb/minitest#specs so the latter example is implied.